.github/workflows/test.yml

```diff
@@ -61,6 +61,69 @@ jobs:
 
       - name: Run just hypothesis tests with more examples
         run: pytest -n 4 tests --disable-warnings -m hypothesis --generate-plots
+          
+  rms:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {
+              name: "RMS 12.1.4, 13.0.3, 13.1.0, 14.1.1",
+              os: ubuntu-20.04,
```

Comment:
Later ubuntu versions don't have Python 3.8.6, the specific minor version that RMS uses